EleksTubeHAX_pio/src/Clock.cpp

```diff
@@ -72,7 +72,7 @@ void Clock::begin(StoredConfig::Config::Clock *config_) {
     Serial.println("Loaded Clock config is invalid, using default.  This is normal on first boot.");
     setTwelveHour(false);
     setBlankHoursZero(false);
-    setTimeZoneOffset(1 * 3600);  // CET
+    setTimeZoneOffset(2 * 3600);  // defaulting CEST
```

Comment:
Because I am living in Berlin and it is CEST here in the moment and after flashing, I wanted to have the "right" time shown ;) And before the button and MQTT stuff was working, I didn't had a chance to set the right timezone.

I also worked on a way to set the timezone without a geolocation needed (also on the TODO list AFAIR), but failed till now and this a rest of it... 

I guess for 90 percent of the human mankind, every timezone you set will be the wrong one in the beginning...So I don't care, which one is set by default...